pkg/adapter/v2/main.go

```diff
@@ -87,6 +88,7 @@ func MainWithContext(ctx context.Context, component string, ector EnvConfigConst
 				}()
 			}
 		}
+		crstatusevent.Config(metricsConfig)
```

Comment:
It's probably cleaner to create a new StatusEvent object instead of relying on global state. Something like statusEvent.New() ... and then pass it to NewcloudEventsClient

Reply:
can you also do the same in main_message.go?